support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/StageStep/help-concurrency.html

```diff
@@ -0,0 +1,7 @@
+<p>
+    Concurrency level.
+</p>
+<p>
+    A concurrency of one is useful to let you lock a singleton resource, such as deployment to a single target server. 
+    Only one build will deploy at a given time: the newest which passed all previous stages
```

Comment:
Again I wonder if this should either link to the [tutorial](https://github.com/jenkinsci/workflow-plugin/blob/master/TUTORIAL.md#stages), or if the details there should be moved here.